### DISCUSSION TITLE
No custom static navigation menus

### DISCUSSION THREAD

**Discussion by Xt0ff** [requarks/wiki#3379] [Help / Questions]
After the update, the static navigation menus that I created are no longer present.
This was already the case during update 2.3.81
If it is not something stored in the database, I have not found where it is ...
What should I do ?

**Comment by NGPixel** [comment]
What do you mean by "no longer present"? Which version are you on?

**Comment by Xt0ff** [comment]
After the update, I no longer have the static menus that I had created.

I am currently in version 2.3.81 and wanted to update to version 2.4.75. I saw that in the meantime there was 2.4.105.
I usually test on a virtual machine before using in production and that's when I realized it.

When I updated from 2.3.77 to 2.3.81, I already had the same problem. Wanting to use it in production I had recreated all the menus ...
